openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts

```diff
@@ -66,3 +68,14 @@ export const getEntityAPIfromSource = (
       return getContainerByName;
   }
 };
+
+export const handleDataAssetAfterDeleteAction = (isSoftDelete?: boolean) => {
+  if (isSoftDelete) {
+    setTimeout(() => {
+      refreshPage();
+    }, 1000);
+  } else {
+    history.push('/');
+    history.go(0);
```

Comment:
Do we need this extra history.go?

Reply:
Pushing the path('/') just updates the url but does not update the page content, so triggering the refresh manually after that